hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotUtils.java

```diff
@@ -93,6 +93,11 @@ public static SnapshotInfo getSnapshotInfo(OzoneManager ozoneManager,
                                              SnapshotChainManager chainManager,
                                              UUID snapshotId) throws IOException {
     String tableKey = chainManager.getTableKey(snapshotId);
+    if (tableKey == null) {
+      LOG.error("Snapshot with ID: '{}' is not found.", snapshotId);
+      throw new OMException("Snapshot with ID: '" + snapshotId + "' is not found.",
```

Comment:
UUID would be printed as a long string so I'd prefer the message itself to be in the front for slightly better readability

```suggestion
      LOG.error("Snapshot not found with UUID '{}'", snapshotId);
      throw new OMException("Snapshot not found with UUID '" + snapshotId + "'",
```

Reply:
done